apps/condo/domains/organization/integrations/sbbol/SbbolCredentials.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path')
+const querystring = require('querystring')
+const conf = require('@core/config')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { getOrganizationAccessToken } = require('./accessToken')
+const { SbbolRequestApi } = require('./SbbolRequestApi')
+const { Organization, TokenSet } = require('@condo/domains/organization/utils/serverSchema')
+const { SBBOL_IMPORT_NAME } = require('./common')
+const dayjs = require('dayjs')
+
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
+
+class SbbolCredentials {
+
+    context = null
+
+    /**
+     *
+     * @param condoEntryPoint path to entrypoint file relative to current directory
+     * @return {Promise<void>}
+     */
+    async connect ({ condoEntryPoint }) {
```

Comment:
why do you need it? It's not a part of this class it's a part of overlying logic

Reply:
I need an instance of Keystone in following places:
- `apps/condo/domains/organization/tasks/refreshSbbolClientSecret.js`
- `apps/condo/bin/sbbol/credentials.js`

Do you know, how to get it there?

Reply:
I need an instance of Keystone in following places:
- `apps/condo/domains/organization/tasks/refreshSbbolClientSecret.js`
- `apps/condo/bin/sbbol/credentials.js`

Will look at `getSchemaCtx`, maybe it will work

Reply:
It seems to be working
[a3f9a00](https://github.com/open-condo-software/condo/pull/911/commits/a3f9a002f24f47ad294be7641f7f6015eaa98c2a)